docs/user-defined-type-functions.md

```diff
@@ -123,55 +122,55 @@ This section details the initial programming interface we propose for `type`s wh
 
 | Instance Methods | Type | Description |
 | ------------- | ------------- | ------------- |
-| `gettype()` | `type` | returns the `type` being negated |
+| `inner()` | `type` | returns the `type` being negated |
 
 #### Singleton
 
 | Instance Methods | Return Type | Description |
 | ------------- | ------------- | ------------- |
-| `getvalue()` | `string \| boolean` | returns the string value of the  singleton |
+| `value()` | `string \| boolean` | returns the string value of the  singleton |
 
 #### Table
 
 | Instance Methods | Return Type | Description |
 | ------------- | ------------- | ------------- |
-| `setprop(key: type, value: type?)` | `nil` | adds / overrides (if same key exists) a key, value pair to the table's properties; if value is nil, removes the key, value pair; if the key does not exist and the value is nil, nothing happens |
-| `getprop(key: type)` | `type?` | returns the value associated with the key from the table's properties; if the key does not exists, returns nil |
-| `getprops()` | `{[type]: type}` | returns a table of the table's properties |
+| `setproperty(key: type, value: type?)` | `nil` | adds / overrides (if same key exists) a key, value pair to the table's properties; if value is nil, removes the key, value pair; if the key does not exist and the value is nil, nothing happens |
+| `getproperty(key: type)` | `type?` | returns the value associated with the key from the table's properties; if the key does not exists, returns nil |
```

Comment:
Most of the API clipped `get` prefix which is a big +1 from me, but this one is an odd duck.

Reply:
I had a comment I was writing earlier (that I'll probably post tomorrow when I'm back on my work PC) about the change, but the lua libraries themselves are inconsistent on this kind of thing, and... frankly, I tried to mirror what I imagine the rationale to be here too. I don't mind calling it `property` too much, but it does feel a little bit off to me because of it having a parameter, I think.